engine/src/main/java/io/zeebe/engine/processor/workflow/deployment/model/element/ExecutableCallActivity.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.engine.processor.workflow.deployment.model.element;
+
+import io.zeebe.util.buffer.BufferUtil;
+import org.agrona.DirectBuffer;
+
+public class ExecutableCallActivity extends ExecutableActivity {
+
+  private DirectBuffer processId;
+
+  public ExecutableCallActivity(String id) {
+    super(id);
+  }
+
+  public DirectBuffer getProcessId() {
```

Comment:
Could we rename this to something like `calledElementId`? Reading `context.getElement().getProcessId()` in the handlers was a little misleading since it's not super obvious we're talking about the called element's processId.

Reply:
I can rename it to `getCalledElementProcessId()`.